src/app/[locale]/(auth)/login/page.tsx

```diff
@@ -11,8 +20,34 @@ export default function LoginPage() {
 
   return (
     <main className="h-full w-full bg-gradient">
-      로그인페이지
-      <button onClick={handleGoBack}>뒤로가기</button>
+      <div>
+        <p className="flex items-center justify-end p-3 pb-0">
+          <RiCloseLargeLine className="mt-[10px] h-6 w-6 cursor-pointer" />
```

Comment:
```suggestion
          <RiCloseLargeLine className="mt-[10px] h-6 w-6 scale-transition-105" />
```

제가 미리 말 못드렸는데, 아이콘들에는 `scale-transition-105` 클래스 넣어서 커서 표시 바꾸기 + hover 시 스케일 업 효과로 통일시키시죠!

Reply:
✅ 89876879a51ebbc1904c38000087abd7942eedda

Reply:
이 부분 다시 수정해주세요.
hover 시에 커지는 효과를 줘야합니다.
다른 아이콘들에 제가 넣어논 것 확인해주세요

scale-transition-105 -> hover:scale-transition-105

Reply:
✅ 8916f28e0a62589b09af82bbc581c80930a9f2c0